src/TestProblem/Hydro/BlastWave/Init_TestProb_Hydro_BlastWave.cpp

```diff
@@ -152,6 +173,14 @@ void SetParameter()
                                                                                        Blast_Center[2] );
 #     ifdef MHD
       Aux_Message( stdout, "  magnetic field strength   = %13.7e\n", Blast_BField );
+      Aux_Message( stdout, "  resetting magnetic field  = %d\n",     OPT__RESET_FLUID );
+      if ( OPT__RESET_FLUID ) {
```

Comment:
Not sure if we also need to include `OPT__RESET_FLUID` in `if (...)`.

Reply:
Should be `OPT__RESET_FLUID_INIT`

Reply:
This test is only for testing `OPT__RESET_FLUID` but not `OPT__RESET_FLUID_INIT`. The latter has no effect here since `dt==0.0`.